scripts/wredstat

```diff
@@ -0,0 +1,413 @@
+        """
+        Get stat for the port
+        If JSON option is True  print data in JSON format
+        """
+        if not port in self.port_queues_map:
+            print("Port does not exist!", port)
+            sys.exit(1)
+
+        # Get stat for the port queried
+        cnstat_dict = self.get_cnstat(self.port_queues_map[port])
+        cnstat_fqn_file_name = cnstat_fqn_file + port
+        json_output = {}
+        json_output[port] = {}
+        if os.path.isfile(cnstat_fqn_file_name):
+            try:
+                cnstat_cached_dict = json.load(open(cnstat_fqn_file_name, 'r'))
+                if json_opt:
+                    json_output[port].update({"cached_time":cnstat_cached_dict.get('time')})
+                    json_output.update(self.cnstat_diff_print(port, cnstat_dict, cnstat_cached_dict, json_opt))
+                else:
+                    print("Last cached time was " + str(cnstat_cached_dict.get('time')))
+                    self.cnstat_diff_print(port, cnstat_dict, cnstat_cached_dict, json_opt)
+            except IOError as e:
+                print(e.errno, e)
+        else:
+            if json_opt:
+                json_output.update(self.cnstat_print(port, cnstat_dict, json_opt))
+            else:
+                self.cnstat_print(port, cnstat_dict, json_opt)
+
+        if json_opt:
+            print(json_dump(json_output))
+
+    def save_fresh_stats(self):
+        # Get stat for each port and save
+        for port in natsorted(self.counter_port_name_map):
+            cnstat_dict = self.get_cnstat(self.port_queues_map[port])
+            try:
+                json.dump(cnstat_dict, open(cnstat_fqn_file + port, 'w'), default=json_serial)
+            except IOError as e:
+                print(e.errno, e)
+                sys.exit(e.errno)
+            else:
+                print("Clear and update saved counters for " + port)
+
+def main():
+    global cnstat_dir
+    global cnstat_fqn_file
+
+    parser  = argparse.ArgumentParser(description='Display the wred queue counters',
+                                      formatter_class=argparse.RawTextHelpFormatter,
+                                      epilog="""
+Examples:
+  wredstat
+  wredstat -p Ethernet0
+  wredstat -c
+  wredstat -d
+""")
+
+    parser.add_argument('-p', '--port', type=str, help='Show the wred queue counters for just one port', default=None)
+    parser.add_argument('-c', '--clear', action='store_true', help='Clear previous stats and save new ones')
+    parser.add_argument('-d', '--delete', action='store_true', help='Delete saved stats')
+    parser.add_argument('-v', '--version', action='version', version='%(prog)s 1.0')
+    parser.add_argument('-j', '--json_opt', action='store_true', help='Print in JSON format')
+    parser.add_argument('-V', '--voq', action='store_true', help='display voq stats')
+    parser.add_argument('-n','--namespace', default=None, help='Display queue counters for specific namespace')
```

Comment:
add a space between `'-n',` and `'--namespace'`

Reply:
Addressed the review comments.!